src/main/java/seedu/address/model/item/Item.java

```diff
@@ -2,43 +2,85 @@
 
 import java.util.Objects;
 
+import java.time.LocalDateTime;
+import java.time.Period;
+
+import seedu.address.commons.util.CollectionUtil;
 import seedu.address.model.tag.UniqueTagList;
 
 /**
  * Represents a Item in the address book. Guarantees: details are present and
  * not null, field values are validated.
  */
-public abstract class Item implements ReadOnlyItem{
+public class Item {
 
 
     private UniqueTagList tags;
 	private Description description;
 	private boolean isDone;
 	
+	private LocalDateTime startDate;
+    private LocalDateTime endDate;
```

Comment:
Oops, re-tabbed and fixed the formatting. I think it was leftover tabs from the original `addressbook`.